services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -137,40 +149,79 @@
+		class="edu.unc.lib.dl.data.ingest.solr.filter.SetAccessControlFilter">
+		<property name="aclFactory" ref="inheritedAclFactory" />
 	</bean>
-	<bean id="setDisplayOrderFilter"
-		class="edu.unc.lib.dl.data.ingest.solr.filter.SetDisplayOrderFilter">
+	<bean id="setAccessStatusFilter"
+		class="edu.unc.lib.dl.data.ingest.solr.filter.SetAccessStatusFilter">
+		<property name="inheritedAclFactory" ref="inheritedAclFactory" />
+		<property name="objectAclFactory" ref="objectAclFactory" />
 	</bean>
-	<bean id="setRelationsFilter"
-		class="edu.unc.lib.dl.data.ingest.solr.filter.SetRelationsFilter">
+	<bean id="setCollectionSupplementalInformationFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetCollectionSupplementalInformationFilter">
+		<property name="collectionFilters" value="${services.indexing.collectionFilters}" />
 	</bean>
-	<bean id="setPathFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetPathFilter">
-		<property name="collectionsPid" ref="collectionsPid" />
-		<property name="tsqs" ref="tripleStoreQueryService" />
+	<bean id="setContentStatusFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetContentStatusFilter">
+	</bean>
+	<bean id="setContentTypeFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetContentTypeFilter">
+	</bean>
+	<bean id="setDatastreamFilter"
+		class="edu.unc.lib.dl.data.ingest.solr.filter.SetDatastreamFilter">
 	</bean>
 	<bean id="setDescriptiveMetadataFilter"
 		class="edu.unc.lib.dl.data.ingest.solr.filter.SetDescriptiveMetadataFilter">
+		<!--  <property name="vocabManager" value="${services.indexing.vocabManager}" />-->
+	</bean>
+	<bean id="setFullTextFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetFullTextFilter">
+	</bean>
+	<bean id="setObjectTypeFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetObjectTypeFilter">
+	</bean>
+	<bean id="setPathFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetPathFilter">
+	   <property name="pathFactory" ref="contentPathFactory" />
 	</bean>
 	<bean id="setRecordDatesFilter"
 		class="edu.unc.lib.dl.data.ingest.solr.filter.SetRecordDatesFilter">
 	</bean>
-	<bean id="setAccessControlFilter"
-		class="edu.unc.lib.dl.data.ingest.solr.filter.SetAccessControlFilter">
-	</bean>
-	<bean id="setContentStatusFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetContentStatusFilter">
+	<bean id="setRelationsFilter"
+		class="edu.unc.lib.dl.data.ingest.solr.filter.SetRelationsFilter">
 	</bean>
-	<bean id="setFullTextFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetFullTextFilter">
-		<property name="accessClient" ref="accessClient" />
+	<bean id="setStatusTagsFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetStatusTagsFilter">
 	</bean>
-	<bean id="setCollectionSupplementalInformationFilter" class="edu.unc.lib.dl.data.ingest.solr.filter.SetCollectionSupplementalInformationFilter">
-		<property name="collectionFilters" value="${services.indexing.collectionFilters}" />
-	</bean>-->
 
 	<!-- Ingest Actions -->
```

Comment:
They aren't going away, they are just part of the user operation based indexing rather than object creation workflow.  I think there's another ticket for that